pkg/api/types.go

```diff
@@ -1920,6 +1920,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// If specified, the pod's tolerations
+	// +optional
+	Tolerations []Toleration
```

Comment:
I think it is better to clarify that the `Tolerations` for a Pod and the  `Taint` for a node should be pair, the `Tolerations` for a pod can only take effect when there are nodes with `Taint`, comments?